overlord/snapstate/handlers_components.go

```diff
@@ -278,7 +278,7 @@ func (m *SnapManager) doLinkComponent(t *state.Task, _ *tomb.Tomb) error {
 	if snapSt.LastCompRefreshTime == nil {
 		snapSt.LastCompRefreshTime = make(map[string]time.Time)
 	}
-	snapSt.LastCompRefreshTime[compSetup.CompSideInfo.Component.ComponentName] = timeNow()
+	snapSt.LastCompRefreshTime[compSetup.ComponentName()] = timeNow()
```

Comment:
is there anywhere else this could be used?

Reply:
I found a couple more places, the issue is that in some cases we store CompSideInfo and use the name from there so usage gets a bit hidden.